FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
-# unreleased
-- [added] Added new API `FIRMessagingExtensionHelper exportDeliveryMetricsToBigQuery` that allows developers to enable notification delivery metrics to BigQuery. (#6181)
+# 2021-07 -- v8.5.0
+- [added] Added new API `FIRMessagingExtensionHelper exportDeliveryMetricsToBigQuery` that allows developers to enable notification delivery metrics that exports to BigQuery. (#6181)
```

Comment:
Perhaps more clear and concise, if it keeps the right meaning?

Added API `FIRMessagingExtensionHelper exportDeliveryMetricsToBigQuery` to enable notification delivery metric exports to BigQuery.